src/EditorFeatures/Core/EditorConfigSettings/Updater/SettingsUpdaterBase.cs

```diff
@@ -16,8 +16,11 @@ namespace Microsoft.CodeAnalysis.Editor.EditorConfigSettings.Updater;
 internal abstract class SettingsUpdaterBase<TOption, TValue> : ISettingUpdater<TOption, TValue>
 {
     private readonly List<(TOption option, TValue value)> _queue = [];
-    private readonly SemaphoreSlim _guard = new(1);
     private readonly IAsynchronousOperationListener _listener;
+#pragma warning disable RS0030 // Do not use banned APIs
```

Comment:
Why would this be configured to ban this in the first place?

Reply:
the JTF analyzers seem to hate it.  But i'm in the position of: that's too bad, we use this everywhere.   :)